test/lib/accounts.spec.js

```diff
@@ -0,0 +1,206 @@
+import * as accounts from '../../src/lib/accounts'
+
+const mockKonnector = {
```

Comment:
I suggest using `konnectorMock` instead, as _mock_ is also a verb, it feels like a function name.